src/domain/artist/generator/artist.generator.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, OnModuleInit } from "@nestjs/common";
+import { Artist } from "../entity/artist.entity";
+import { InjectRepository } from "@nestjs/typeorm";
+import { Repository } from "typeorm";
+
+@Injectable()
+export class ArtistGenerator implements OnModuleInit {
+  constructor(
+    @InjectRepository(Artist)
+    private readonly artistRepository: Repository<Artist>
+  ) {}
+
+  async onModuleInit() {
+    const artists = [
+      "스텔라이브",
+      "아야츠노 유니",
+      "아이리 칸나",
+      "시라유키 히나",
+      "아라하시 타비",
+      "네네코 마시로",
+      "아카네 리제"
+    ];
+
+    artists.forEach(async (artist) => {
+      if (!(await this.artistRepository.existsBy({ name: artist })))
```

Comment:
```suggestion
      const isExist = await this.artistRepository.existsBy({ name: artist })
      if (!isExist)
```
이거 어때요?
괄호가 너무 많아지는거같아서 변수 추출 해봤어용

Reply:
좋은거 같아요!